.travis.yml

```diff
@@ -36,6 +39,7 @@ addons:
     username: "amphtml"
   jwt:
     secure: "Wze0F0vGL0UcxryOx1n/vcuD5LIMGyR+69Nc6IWLoRvZBbbIpFwVFhDE6rE9ranIXiA2Hc684N4sV8ASfNDF8RRSB+jyLov159qwgji2rBxIfQ/4kuDV2vYoAJvYMz8m42kwx5FV2VV9awqMMt8mwU3wYIrKIaVCxB34uV86KIlDlbrHxt17Bm5EIiUmwi9r1AAnW/63vVRUN264D77oB4j9UQ759PfD6BDwEt54O87KurNIaLseNCr1IvzfL8veEsZ3uTbLC1GtgHfR4IGgkS2YyN2QIk06VZWeRDEOalS3RcY0nDkbCmBywxIGObnrpEMzOpjBiOb2fxLoLvvpjlla5W84zJGfWE6q4T9IvkyHuDJE+sft5B+arjMIeA6PIeUhKdV27+6qqDEf7fILZ/U/Ekn9ds4zSV8hekAZPUyyPncOeyWppCIJ8sOeCrsebkRjH1BoX/d+FE+nP0bN/XkBpIi/nManx5FyS/kqjQWGKmvsFQfEWlSUaZi7XtEQEjvBizRkzvpJanSDaoiTDS2Keulmwii3XRId51FuGtnfDZFeggLaMTKGfBX9DlPkccwYAZe6vPNfYk1pNgEj6AtnifEhYVEO+aAuWhEnJ86od+1wDOL/h+a2XY6h8/gFBywsD95p7sXPfdVDCKgwagiBo+Hw5MNjztVF7lszg1A="
+    secure: "WJxhl/duuCxQW0C5pgpWgrFOvpGabCKhi5IKnyUtaLiWtMXBx4XMYh0ou60zfk5h76cgmXbHcKpHoo6Gyd0enxnwLcnrgP5RdyXPq4lOK8+k/cNNQpXx7VqQl37UohBWeZrEkIFBNOjohlK4jvrOgE4QmZuKwm14dw++CNEI4yNVliLgE6bGyFeaLrVyJufGwDkWKm5aZcq79i+g6BfxYZ5X+otL0XFxaLcWCUUm004taPMYSd1BpZVNuoTnyRsmhSz+wbBdILj7SQMqAj3XIAHAHE4SG9rT7u7Kt236DlAyiIppIL30uttGlS/AHPFpORE6b+kVgO4Xzf4REoGqbK23+TVsdEQJ63NXQcjmFgu9tgNy4+mZ5NDd1rP5zoZTBlo/Co989RgM+gYHJOilc1HYkGOy30O7Im/7FjWPdFmGVmzSYpFPG8zNWM+mNk1Cn74ueyF3GuDw+G5DtVEzwlP8n4qUnVh2TGNkZ8gYlFeUDL0CovvsYPivdK9p2ZQqoLPNKlPC1AqKlo0dyF702NTn36X99q3fMJ/EmspPbaj34Peno1oIAfF4s8zne3l7PiSkAFCFCBUCmyqvwHlvLHJLNtGMf+RG5tlRNZbS7iyW6cQHfftnJrqXuo7cajLU9/5e7qJnyTbu+AUdL+oCYScVU8GQBUHRuPxMHgBZfvs="
```

Comment:
@cramforce This line is a placeholder, and is the result of `travis encrypt GH_TOKEN=foobar`. I'll need to replace it with the encrypted version of the real `GH_TOKEN`, generated as described here: https://github.com/greenkeeperio/greenkeeper-lockfile#setup